gpgme.gemspec

```
@@ -17,7 +17,7 @@ Made Easy). GnuPG Made Easy (GPGME) is a library designed to make access to
 GnuPG easier for applications. It provides a High-Level Crypto API for
 encryption, decryption, signing, signature verification and key management.}
 
-  s.add_runtime_dependency "mini_portile2", "~>2.3"
+  s.add_runtime_dependency "mini_portile2", "~>2.7.0"
```

Is there any reason this couldn't be:

```suggestion
  s.add_runtime_dependency "mini_portile2", "~>2.7"
```

Thanks!

---

I think #163 covers this as well. We might need a new release on rubygems.org though.

---

@ueno - nice catch, thanks! Looks like `2.0.22` should cover it: https://github.com/ueno/ruby-gpgme/commit/b8af6be305d7b8bf11a17c87bee4be0c41f4088d

Will give that a shot, thanks!